src/connectors/hits/connectHits.ts

```diff
@@ -92,10 +120,14 @@ const connectHits: HitsConnector = function connectHits(
           typeof escapeHits
         >).__escaped = initialEscaped;
 
+        sendEvent('view', results.hits);
+
         renderFn(
           {
             hits: results.hits,
             results,
+            sendEvent,
```

Comment:
What happens when `sendEvent` is called without the Insights middleware?

Reply:
`sendEvent` eventually calls `instantSearchInstance.sendEventToInsights()` and it's `noop` by default. (https://github.com/algolia/instantsearch.js/blob/feat/send-events/src/lib/InstantSearch.ts#L245:L245)